cross-project-tests/dtlto/dtlto.c

```diff
@@ -0,0 +1,35 @@
+// REQUIRES: x86-registered-target,ld.lld
+
+/// Simple test that DTLTO works with a single input bitcode file and that
+/// --save-temps can be applied to the remote compilation.
+// RUN: rm -rf %t && mkdir %t && cd %t
+
+// RUN: %clang --target=x86_64-linux-gnu -c -flto=thin %s
+
+// RUN: ld.lld dtlto.o \
+// RUN:   --thinlto-distributor=%python \
+// RUN:   --thinlto-distributor-arg=%llvm_src_root/utils/dtlto/local.py \
+// RUN:   --thinlto-remote-compiler=%clang \
+// RUN:   --thinlto-remote-compiler-arg=--save-temps
+
+/// Check that the required output files have been created.
+// RUN: ls | count 10
```

Comment:
`ls | sort | FileCheck %s` then we can drop DAG and the `count 10` test with `CHECK-NOT: {{.}}`

We can use `sort` in testsuite, `rg 'RUN:.* sort '`

Reply:
Nice suggestion - thanks :) I have added a commit to address this.